### PR TITLE
home page done

### PR DESCRIPTION
**Pull request by TJain02**
Netlify preview: https://e-com-home-page.netlify.app/

### CODE REVIEW

File: CSS-Components/styles.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  display: grid;
+  grid-template-columns: auto auto;
+  margin: 1rem;
+  column-gap: 1rem;
+}
+
+.shoe-card{
+  margin: 0.3rem;
+  display: flex;
+  background-color: #eef2ff;
```
Comment (harshitpaliwal95) [inline CSS-Components/styles.css]: use color variables

**Review by harshitpaliwal95 — COMMENTED** [review]
overall locks good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </div>
+
+    </div>
+
+    <div class="main-section">
+
+        <!-- <br>
+        <br>
+        <br> -->
+
```
Comment (DevNirah7) [inline index.html:43]: Can delete these comments

**Review by DevNirah7 — APPROVED** [review]
Can use color variables and remove commented block specified. Rest all looks good to commit.

**Pull request merged**